app/api/waitlists/[id]/users/route.ts

```diff
@@ -85,3 +86,109 @@ export const GET = async (
     powerBadgeUsersCount: powerBadgeCount,
   });
 };
+
+export const POST = async (
+  req: NextRequest,
+  {
+    params: { id },
+  }: {
+    params: { id: string };
+  }
+) => {
+  try {
+    // Find the waitlist associated with the id
+    const waitlist = await prisma.waitlist.findUnique({
+      where: {
+        id: parseInt(id),
```

Comment:
we also need to check if the caller address is the owner of the waitlist

Reply:
![image](https://github.com/builders-garden/beearly/assets/140813289/40a8892d-4084-4cb2-a05a-b88e4bbb8dac)